data-raw/ni-beds.R

```diff
@@ -0,0 +1,40 @@
+# Northern Ireland on Health Inequalities Explorer
+# Secondary care indicators - bed availability
+
+# ---- Load libs ----
+library(tidyverse)
+library(devtools)
+
+# ---- Load internal sysdata.rda file with URL's ----
+load_all(".")
+
+query_url <-
+  query_urls |>
+  filter(id == "ni_beds") |>
+  pull(query)
+
+# ---- Download and wrangle data ----
+raw <- read_csv(query_url, skip = 2)
+
+# Removed "Day Case" column since it included missing values and duplicate values with "Total Day Cases".
+ni_beds <- raw |>
+  select(
+    quarter_ending = "Quarter Ending",
+    trust_code = "HSC Trust",
```

Comment:
`"HSC Trust"` isn't a code, it is a name. The trust codes you need can be found in `geographr::boundaries_trusts_ni18`:

<img width="1040" alt="Screenshot 2023-07-11 at 10 53 07" src="https://github.com/humaniverse/healthyr/assets/38110953/fa6272cf-054a-447b-b1c7-05e5131c8481">

To add these codes to this data set you will need to create a lookup like so:

```r
library(sf)

lookup_trusts <- geographr::boundaries_trusts_ni18 |> 
  st_drop_geometry()

```
and then join this lookup to the tibble you have created. Please make the `trust18_code` the first column in the data set. Please drop the `trust18_name` column, as just the trust code is needed as a primary key to join to other data sets in the humaniverse and HIE.

Reply:
I have made these changes! Please let me know if there is anything else I need to do :)

Reply:
Don't forget to hit "Revolve conversation" when you think your changes are ready! I will do it now.